src/features/unified-share-modal/UnifiedShareModal.js

```diff
@@ -41,11 +41,13 @@ class UnifiedShareModal extends React.Component<USMProps, State> {
     constructor(props: USMProps) {
         super(props);
 
+        const { initialDataReceived } = props;
+
         this.state = {
-            getInitialDataCalled: false,
+            getInitialDataCalled: !!initialDataReceived,
             isConfirmModalOpen: false,
             isEmailLinkSectionExpanded: false,
-            isFetching: true,
+            isFetching: !initialDataReceived,
```

Comment:
If initial-data was received, I'd expect `isFetching` to be false. Unless I'm missing something?

Reply:
Yes, that's correct. When the ContentSharing element instantiates the USM, it passes `initialDataReceived` as a prop, so `isFetching` will be false.